### ISSUE TITLE
[Bug] Minor update required in CLI binary source file's path name. 

### ISSUE THREAD

**Issue by prady0t** [kyverno/website#917]
### Page link

https://kyverno.io/docs/kyverno-cli/

### Description

Documentation states that to build CLI from source use this command: 
```
make build-cli
mv ./cmd/cli/kubectl-kyverno/kyverno /usr/local/bin/kyverno
```
but binary file generated is named as : `kubectl-kyverno`  
![Screenshot 2023-07-14 at 2 42 50 PM](https://github.com/kyverno/website/assets/99216956/52af9d63-f462-49c4-93fa-59c4cc1728c5)

Hence above command throws error:  ***mv: ./cmd/cli/kubectl-kyverno/kyverno: No such file or directory***

### Expected behavior

Build instruction command should be changed to:
```
make build-cli
mv ./cmd/cli/kubectl-kyverno/kubectl-kyverno /usr/local/bin/kyverno
```

**Comment by prady0t** [comment]
I can see the correct line in the website's source code but on the actual website, it's different. https://github.com/kyverno/website/blob/aae46581c34f0b22b2605fe80e0045d5bdb89bad/content/en/docs/Kyverno%20CLI/_index.md?plain=1#L58
Can someone please explain why is that so?

**Comment by chipzoller** [comment]
Closed in #919
